packages/mobile/src/screens/form-screen/FormScreen.tsx

```diff
@@ -38,7 +38,7 @@ export const FormScreen = (props: FormScreenProps) => {
 
   const handleSubmit = useCallback(() => {
     if (!stopNavigation) {
-      navigation.goBack()
+      navigation.pop()
```

Comment:
This should be safe and is more semantically correct to what is happening.

Reply:
Doesn't this assume we're in a stack navigator? It's probably a fairly safe assumption (especially now)... don't know if it's worth the semantics though